diesel/src/pg/connection/raw.rs

```diff
@@ -98,6 +98,10 @@ impl RawConnection {
         );
         RawResult::new(ptr, self)
     }
+
+    pub unsafe fn transaction_status(&self) -> PGTransactionStatusType {
```

Comment:
There is no need to mark this function as unsafe. As far as I see it's valid to call this function as long as `PgConnection` exists, because `internal_connection` is guaranteed to be not null.